osu.Game/Beatmaps/BeatmapManager.cs

```diff
@@ -68,9 +69,11 @@ public partial class BeatmapManager : DownloadableArchiveModelManager<BeatmapSet
         private readonly RulesetStore rulesets;
         private readonly BeatmapStore beatmaps;
         private readonly AudioManager audioManager;
-        private readonly TextureStore textureStore;
+        private readonly LargeTextureStore largeTextureStore;
         private readonly ITrackStore trackStore;
 
+        private readonly GameHost host;
```

Comment:
As per default ctor argument, this is a potentially

```suggestion
		[CanBeNull]
        private readonly GameHost host;
```

field.